### PR TITLE
Fix linux package docs [skip-ci]

### PR DESCRIPTION
**Pull request by palango**
Update the linux docs for the pyinstaller binary.

### CODE REVIEW

File: docs/overview_and_guide.rst [hunk 1]
```diff
@@ -20,13 +20,13 @@ The preferred way to install Raiden is downloading a self contained application
 Linux
 *****
 
-Download the latest :code:`raiden--x86_64.AppImage`, and make it executable:
```
Comment (LefterisJP) [inline docs/overview_and_guide.rst]: Can't you use the alias we introduced for version here? `|version|`

File: docs/overview_and_guide.rst [hunk 2]
```diff
@@ -20,13 +20,13 @@ The preferred way to install Raiden is downloading a self contained application
...
+Download the latest :code:`raiden-<version>-linux.tar.gz`, and extract it:
 
-    chmod +x raiden--x86_64.AppImage
+    tar -xvzf raiden-<version>-linux.tar.gz
```
Comment (LefterisJP) [inline docs/overview_and_guide.rst]: here too

**Comment by palango** [comment]
@LefterisJP This seems to work as long as it's not included in another environment:


screen shot 2018-02-19 at 14 53 53

**Comment by palango** [comment]
It doesn't seem to be too easy: https://github.com/sphinx-doc/sphinx/issues/4054

I rolled back the changes, lets look at that later.

**Pull request merged**